src/Navigation/WPF/Navigation.WPF/Service/NavigationService.show.cs

```diff
@@ -299,5 +270,24 @@ protected async Task<TViewModel> InitializeViewModel<TViewModel>()
                 viewModel,
                 content);
         }
+
+
+
+        private static TView GetContentInstance<TView>(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to have parameters matching all the type parameters.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5575746)